crates/shared/src/order_quoting.rs

```diff
@@ -702,6 +709,48 @@ fn quote_kind_from_signing_scheme(scheme: &QuoteSigningScheme) -> QuoteKind {
     }
 }
 
+mod metrics {
+    use super::FindQuoteError;
+
+    #[derive(prometheus_metric_storage::MetricStorage)]
+    pub struct Metrics {
+        /// Counter for whether quotes where found by ID, fuzzy search or not
+        /// found at all and for what reason.
+        #[metric(name = "orderbook_quote_finding", labels("success", "type"))]
```

Comment:
I think the name is not needed here. Also metrics automatically get a prefix depending on the pod they get queried in so the `orderbook_` prefix is not needed.